test/Api.Test/Utilities/EnumMatchesAttributeTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Bit.Api.Utilities;
+using Bit.Core.Enums;
+using Xunit;
+
+namespace Bit.Api.Test.Utilities;
+
+public class EnumMatchesAttributeTests
```

Comment:
This validation attribute allows you to check whether the attributed `Enum` property is in a list of values.